Makefile.build

```diff
@@ -1,389 +0,0 @@
-ifeq (,$(BEFORE))
-print-pretty-single-time-diff::
-	@echo 'Error: Usage: $(MAKE) print-pretty-single-time-diff AFTER=path/to/file.v.after-timing BEFORE=path/to/file.v.before-timing'
-	$(HIDE)false
-else
-ifeq (,$(AFTER))
-print-pretty-single-time-diff::
-	@echo 'Error: Usage: $(MAKE) print-pretty-single-time-diff AFTER=path/to/file.v.after-timing BEFORE=path/to/file.v.before-timing'
-	$(HIDE)false
-else
-print-pretty-single-time-diff::
-	$(HIDE)$(COQMAKE_BOTH_SINGLE_TIMING_FILES) --fuzz=$(TIMING_FUZZ) --sort-by=$(TIMING_SORT_BY) $(TIMING_USER_ARG) $(AFTER) $(BEFORE) $(TIME_OF_PRETTY_BUILD_FILE) $(TIME_OF_PRETTY_BUILD_EXTRA_FILES)
-endif
-endif
-pretty-timed:
-	$(HIDE)$(MAKE) --no-print-directory make-pretty-timed
-	$(HIDE)$(MAKE) --no-print-directory print-pretty-timed
-.PHONY: pretty-timed make-pretty-timed make-pretty-timed-before make-pretty-timed-after print-pretty-timed print-pretty-timed-diff
-
-ifneq (,$(TIMING))
-TIMING_ARG=-time
-ifeq (after,$(TIMING))
-TIMING_EXT=after-timing
-else
-ifeq (before,$(TIMING))
-TIMING_EXT=before-timing
-else
-TIMING_EXT=timing
-endif
-endif
-else
-TIMING_ARG=
-endif
-
-###########################################################################
-# Build the .v deps
-VFILED=.vfiles.d
--include $(VFILED)
-
-# All dependency includes must be declared secondary, otherwise make will
-# delete them if it decided to build them by dependency instead of because
-# of include, and they will then be automatically deleted, leading to an
-# infinite loop.
-
-.SECONDARY: $(VFILED)
-
-###########################################################################
-# Compilation options
-###########################################################################
-
-# Default timing command
-# Use /usr/bin/env time on linux, gtime on Mac OS
-TIMEFMT?="$@ (real: %e, user: %U, sys: %S, mem: %M ko)"
-ifneq (,$(TIMED))
-ifeq (0,$(shell /usr/bin/env time -f "" true >/dev/null 2>/dev/null; echo $$?))
-STDTIME?=/usr/bin/env time -f $(TIMEFMT)
-else
-ifeq (0,$(shell gtime -f "" true >/dev/null 2>/dev/null; echo $$?))
-STDTIME?=gtime -f $(TIMEFMT)
-else
-STDTIME?=time
-endif
-endif
-else
-STDTIME?=/usr/bin/env time -f $(TIMEFMT)
-endif
-
-TIMER=$(if $(TIMED), $(STDTIME), $(TIMECMD))
-
-# NB: do not use a variable named TIME, since this variable controls
-# the output format of the unix command time. For instance:
-#   TIME="%C (%U user, %S sys, %e total, %M maxres)"
-
-PLUGININCLUDES=$(addprefix -I _build/default/, $(wildcard plugins/*))
-DUNEPLUGININCLUDES=$(addprefix -I _build/default/, $(wildcard plugins/*))
-NATIVEINCLUDES=$(addprefix -nI _build/default/, kernel/.kernel.objs/byte)
-
-ifdef NATIVECOMPUTE
-  COQOPTS += -w -deprecated-native-compiler-option -native-compiler ondemand
-endif
```

Comment:
What's the correct flag to now pass for native?

Reply:
The reason I am asking is because the native build was one of the sore spots of the dune transition. We need to make sure that absolutely everything done before is supported.

Reply:
Not sure what you mean. `-native-compiler foo` is deprecated since the introduction of `coqnative`, however that method is much slower.

Reply:
Yes as of now it is not removed, and given a 33% increase of build time, we may need to tweak coqnative more before we go ahead.